vmdb/config/environments/production.rb

```diff
@@ -3,6 +3,7 @@
 
   # Code is not reloaded between requests
   config.cache_classes = true
+  config.eager_load = true
```

Comment:
This should not be true.  It is intentionally false for our app (at least for now / until we get to forking workers) cc @jrafanie